source/base/Text.ooc

```diff
@@ -149,6 +149,30 @@ Text: cover {
 			--rightPosition
 		this slice(leftPosition, rightPosition - leftPosition + 1)
 	}
+	replaceAll: func (toReplace, newValue: This) -> This {
+		t := this take()
+		result: This
+		position := t find(toReplace)
+		if (position == -1)
+			result = t copy()
+		else {
+			previous := 0
+			result = This new()
+			x := toReplace take()
+			y := newValue take()
```

Comment:
I saw that variables with one letter name are used in this file in other methods.